templates/home.html

```diff
@@ -128,6 +139,11 @@ <h2>{{ filter_message }}</h2>
                     }
                 }
             }
+
+            function rollDice() {
+                let rollDiceValue = document.getElementById("roll-dice").value;
+                window.location.search += "&roll-dice=" + rollDiceValue;
```

Comment:
Be careful with "+" sign in the text field. It will get it as an extra parameter instead of only one parameter

Reply:
this line also adds infinite times the "roll-dice" parameter to the url if pressed multiple times.

Reply:
Adding only one parameter would remove all filtering, creating a "weird" user experience